blog/2024-05-13-april-and-may-releases.mdx

```diff
@@ -0,0 +1,216 @@
+---
+title: "April and May 2024 Releases"
+slug: "2024-05-13-april-and-may-releases"
+authors: [beeankha]
+tags: [announcement, conda, conda-build]
+description: |
+  Conda 24.4.0, conda 24.5.0, and conda-build 24.5.0 have been released! 🎉
+image: img/blog/2024-05-13-april-and-may-releases/banner.png
+---
+
+The April and May 2024 releases included updates to conda and conda-build! 🎉 All of these new releases have been released to both `main` and `conda-forge`.
+
+## Changes in Conda [24.4.0](https://github.com/conda/conda/releases/tag/24.4.0)/[24.5.0](https://github.com/conda/conda/releases/tag/24.5.0)
+
+To update conda to the latest version, run:
+
+```bash
+conda install -n base conda=24.5.0
+```
+
+<!-- truncate -->
+
+Conda's release cycle is typically every two months per [CEP 8](https://github.com/conda/ceps/blob/main/cep-8.md); however, when a new standalone
+feature is added in between the bimonthly cadence, a separate release is published with the new changes. The April 2024 release of conda introduces
+a new enhancement for Windows users. Specifically, the stub executables used for Python entrypoints in packages are now codesigned as a result of
+[pull request #13721](https://github.com/conda/conda/pull/13721).
```

Comment:
I read this as "when a new standalone
feature is added...a separate release is published". This was a little too strongly worded/lacked context IMO:

```suggestion
Per conda's release schedule ([CEP 8](https://github.com/conda/ceps/blob/main/cep-8.md)) there is an expected and official release every two months;
occasionally however, new  features may be added that are significant enough to warrant a separate release in between the bimonthly cadence. The
April 2024 release of conda introduces a new enhancement for Windows users. Specifically, the stub executables used for Python entrypoints in
packages are now codesigned as a result of [pull request #13721](https://github.com/conda/conda/pull/13721).
```